src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,125 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+    /** Scanner to scan in user input */
+    private Scanner sc;
+
+    /**
+     * Constructor for a UI.
+     */
+    public Ui() {
+        sc = new Scanner(System.in);
+    };
+
+    /**
+     * UI when a task has been added.
+     *
+     * @param taskList The TaskList.
+     * @param task The Task that has been added.
+     */
+    public void addUi(TaskList taskList, TaskList.Task task) {
+        System.out.println("Got it. I've added this task:\n" + "  " + task + "\n" +
+                "Now you have " + taskList.size() + " tasks in the list.");
+    }
+
+    /**
+     * UI when a task has been marked as done.
+     *
+     * @param task The Task that has been marked as done.
+     */
+    public void doneUi(TaskList.Task task) {
```

Comment:
Another issue is that 

> Names representing methods must be verbs and written in camelCase.

Would it be better to rename the methods in the following way?
```suggestion
    public void displayDoneUi(TaskList.Task task) {
```

This issue applies to several other Ui methods within this file.